cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -1346,7 +1280,7 @@ void setWorkloadIdentityProvider(String workloadIdentityProvider) {
   }
 
   @VisibleForTesting
-  void setWorkloadIdentityServiceAccountTokenTTLSeconds(long workloadIdentityServiceAccountTokenTTLSeconds) {
+  void setWorkloadIdentityServiceAccountTokenTTLSeconds(String workloadIdentityServiceAccountTokenTTLSeconds) {
```

Comment:
why do we want to change this?

Reply:
I moved the default TTL logic to the static function in WorkloadIdentityUtil. As such, I just wanted to directly pass in a null string if the config isn't set which would save on code complexity.

Reply:
I can't find where this value is being used, but we generally want to use the correct type to remove the need of validating values at multiple points in the code.

Reply:
This would require validation in both KubeTwillPreparer and KubeMasterEnvironment. Went ahead and changed it back to a long, but introduced another helper method.